CONTRIBUTE.adoc

```diff
@@ -29,3 +29,47 @@ Don’t get discouraged if you don't get an immediate response, this is a side-p
 
 Testing existing features is a good way to contribute to the project, too!
 
+
+== Benchmarking
+
+This project uses [Ginkgo](http://onsi.github.io/ginkgo/) as the main testing framework but uses the standard Go toolchain for benchmarking and performance comparisons.
+
+Since we use Ginkgo, it's generally a good idea to skip the normal tests when running benchmarks.
+To do that, use something like `go test -run=XXX` to skip normal tests (assuming no test contains "XXX").
+
+It's also a good idea to run multiple iterations of the benchmarks to avoid anomolies and gather better statistics.
+
+Example command:
+
+```
+go test -run=XXX -bench=.* -benchmem -count=10 ./... | tee bench-branch1.txt
```

Comment:
the `-bench=.*` flag gives me the following error:
```
zsh: no matches found: -bench=*
```
but `-bench=.` works, instead.